src/AbortableThread.cpp

```diff
@@ -0,0 +1,121 @@
+//+=============================================================================
+//
+// file :         AbortableThread.cpp
+//
+// description :  A thread that run code in a loop till it is aborted
+// project :      TANGO Device Server
+//
+// $Author: graziano $
+//
+// $Revision: 1.6 $
+//
+// $Log: CheckPeriodicThread.cpp,v $
+//
+//
+//
+// copyleft :     European Synchrotron Radiation Facility
+//                BP 220, Grenoble 38043
+//                FRANCE
+//
+//-=============================================================================
+
+
+#include <AbortableThread.h>
+#include <cmath>
+#include "Consts.h"
+
+namespace HdbEventSubscriber_ns
+{
+
+    //=============================================================================
+    //=============================================================================
+    AbortableThread::AbortableThread(Tango::DeviceImpl *dev): Tango::LogAdapter(dev)
+                                                              , abort_flag(false)
+                                                              , abort_condition(&abort_mutex)
+    {
+    }
+
+    //=============================================================================
+    //=============================================================================
+    AbortableThread::~AbortableThread() = default;
+    /*
+    {
+       // join(nullptr);
+    }
+    */
+
+    //=============================================================================
+    //=============================================================================
+    auto AbortableThread::run_undetached(void */*unused*/) -> void *
```

Comment:
GH doesn't like `*/*`, should we fix it or it is irrelevant since moving to GitLab?
Note that there are also others around in the code.

Reply:
if renamed to * /*unused*/ it is ok no ? we did this in some places, I'll try and fix them all, or remove them (I think it is to comply to some clang-tidy, nothing really mandatory here)

Reply:
Yes, just a matter of adding a space. Just to not see these red highlights.

Reply:
done.